### ISSUE TITLE
give up tabs logic from components, use a tabs "widget" in sidebar

### ISSUE THREAD

**Issue by xlotlu** [EEA-Norway-Grants/dataviz#105]
The sidebar shouldn't have any tab-related logic, and neither should the sub-components. We should end up with something like this in the main template:

```html
<aside>
  <sidebar>
    <!-- this entire part is the sidebar's slot... -->
    <tab-group>
      <tab title="Results">
        <results />
      </tab>
      <tab title="Programmes">
        <programmes />
      </tab>
    </tab-group>
    <!-- ...end slot -->
  </sidebar>
</aside>
```
Before we start writing our own tab component, evaluate this project, it looks very much ok: https://github.com/spatie/vue-tabs-component

(It's also debatable if the sidebar has any business being a separate component, but we'll see about that later.)

**Comment by xlotlu** [comment]
@mihai-macaneata we need this to happen soon to avoid duplicated work

**Comment by xlotlu** [comment]
closed by #136
